package.json

```diff
@@ -1,28 +1,42 @@
 {
   "name": "linter-rubocop",
-  "main": "./lib/index.coffee",
-  "version": "0.5.3",
+  "main": "./src/index.js",
+  "version": "0.6.0",
   "description": "Lint Ruby on the fly, using rubocop",
   "repository": "https://github.com/AtomLinter/linter-rubocop",
   "license": "MIT",
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": ">=1.4.0 <2.0.0"
+  },
+  "configSchema": {
+    "command": {
+      "type": "string",
+      "default": "rubocop",
+      "description": "\n This is the absolute path to your `rubocop` command. You may need to run \n `which rubocop` or `rbenv which rubocop` to find this. Examples: \n `/usr/local/bin/rubocop` or `/usr/local/bin/bundle exec rubocop --config \n /my/rubocop.yml`. \n"
+    },
+    "disableWhenNoConfigFile": {
+      "type": "boolean",
+      "title": "Disable when no .rubocop.yml config file is found",
+      "default": false,
+      "description": "\n Only run linter if a RuboCop config file is found somewhere in the path \n for the current file."
+    }
   },
   "activationHooks": [
     "language-ruby:grammar-used",
     "language-ruby-on-rails:grammar-used",
     "language-chef:grammar-used"
   ],
   "scripts": {
-    "lint": "coffeelint lib && eslint .",
+    "lint": "eslint .",
     "test": "apm test"
   },
   "dependencies": {
     "atom-linter": "^9.0.0",
-    "escape-html": "^1.0.3"
+    "escape-html": "^1.0.3",
+    "pluralize": "^3.1.0",
+    "tmp": "0.0.31"
```

Comment:
Any reason to pin this? Greenkeeper will trigger a build for all new versions checking if it causes any issues.